client/rest-high-level/src/test/java/org/elasticsearch/client/IngestClientIT.java

```diff
@@ -93,31 +90,25 @@ public void testDeletePipeline() throws IOException {
     public void testSimulatePipeline() throws IOException {
         XContentType xContentType = randomFrom(XContentType.values());
         XContentBuilder builder = XContentBuilder.builder(xContentType.xContent());
-        int numDocs = randomIntBetween(1, 10);
         boolean isVerbose = randomBoolean();
```

Comment:
can we also have two separate tests for with and without verbose rather than randomizing it?